addon/services/modal.js

```diff
@@ -37,6 +40,8 @@ export default class ModalService extends Service {
 				deferred.reject(null, `Modal: closing '${modal.fullname}'`);
 			}
 		});
+
+		this.trigger('close', key);
```

Comment:
This `close` should be trigger for every modal closed (inside `forEach`) with a modal instance/name as argument.